packages/angular/generators.json

```diff
@@ -115,6 +115,16 @@
       "schema": "./src/generators/scam/schema.json",
       "description": "Generate a component with an accompanying Single Component Angular Module (SCAM)."
     },
+    "scam-directive": {
```

Comment:
Both of these need to be added to the generators block of this JSON, pointing to the non-compat file.

Take a look at the `scam` entry further in the file to get an idea of how it's done.